frontend/src/components/PlayComponents/BetCard.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components/macro';
+
+export default function BetCard({ bet, minBet, cash }) {
+  const [betValue, setBetValue] = useState(minBet);
+  const [betTooSmall, setBetTooSmall] = useState(false);
+  const [betToLarge, setBetTooLarge] = useState(false);
+
+  useEffect(() => {
+    //is this necessary, because otherwise it would not update when the minBet and cash change?
```

Comment:
Yes, this should be right. The initial State minBet of betValue should be set when rendering.

If there is a initial state hardcoded in row 3 it would work without useEffect.